packages/lib/src/services/test/player-sdk.service.spec.ts

```diff
@@ -457,6 +457,26 @@ describe('Service', () => {
 
         expect(callback).not.toHaveBeenCalled();
       });
+
+      it('should match escaped and unescaped URLs', async () => {
```

Comment:
Move it inside `describe('constructor')` as we already have some logic regarding checking the iframe there.